### DISCUSSION TITLE
Microdata with `TechArticle` schema for MDN pages

### DISCUSSION THREAD

**Discussion by kael** [mdn/mdn-community#112] [Content]
Currently MDN pages contain Microdata for the https://schema.org/BreadcrumbList schema.

It'd be nice to have Microdata with the https://schema.org/TechArticle schema, similar to Javascript.info articles ([e.g](https://javascript.info/manuals-specifications)).

```xml
<article class="formatted" itemscope itemtype="http://schema.org/TechArticle">
  <meta itemprop="name" content="Manuals and specifications">
    <div itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="email" content="...@gmail.com">
      <meta itemprop="name" content="Ilya Kantor">
    </div>
    <div itemprop="articleBody">
      ...
    </div>
  </meta>
</article>
  ```

**Comment by teoli2003** [comment]
I'm all for more structure in our pages and microdata may be a way to convey some of it. 

As this looks like a non-trivial amount of work, and before digging into the details about the actual work needed, I'm curious to know if there is a concrete use case for it. 

Are there consumers of such microdata out there able to consume MDN data thanks to this work?

**Reply by kael** [reply]
> Are there consumers of such microdata out there able to consume MDN data thanks to this work?

Well, apart from SEO, I don't know how microdata are consumed.

In my case, I'm using microdata for content classification for my bookmarks (with a tool I wrote).

When bookmarking several pages on the same topic, I then get a bundle of complementary publications with various schema, like `DOAP:specification`, `schema:TechArticle`, `schema:Article`, `schema:QAPage`, `schema:SoftwareSourceCode` or `schema:Code`. And I then can browse content by their schema type.

But yes, as microdata are hidden data their usefulness is not obvious.

Just that since MDN pages contain `schema:BreadcrumbList` (I don't find that schema useful in general, though), I was thinking adding `schema:TechArticle` would be coherent.
